info-commands.go

```diff
@@ -236,6 +247,8 @@ type InfoMessage struct {
 	Services     Services           `json:"services,omitempty"`
 	Backend      interface{}        `json:"backend,omitempty"`
 	Servers      []ServerProperties `json:"servers,omitempty"`
+
+	Pools map[int]map[int]ErasureSetInfo `json:"pools,omitempty"`
```

Comment:
`map[int]` cannot be used in JSON. It must be `map[string]` or arrays.

Reply:
doesn't go automatically convert this to string? @klauspost